app/views/media_objects/show.html.erb

```diff
@@ -26,5 +26,12 @@ Unless required by applicable law or agreed to in writing, software distributed
   </h1>
 </div>
 
-<%= render 'administrative_links' %>
+<div class="row">
+  <% if can? :update, @media_object %>
+    <div class="col-sm-5 mb-3">
```

Comment:
The `can?` check is already in the `administrative_links` partial so how about pushing the `div` into that partial?  Then nothing needs to wrap the render of `administrative_links`.  This would also mirror how the `destroy_checkout` partial starts with an ability check and surrounding `div`.